docs/yaml-config.rst

```diff
@@ -154,6 +154,21 @@ the highest supported minor version will be selected.
     python:
        version: 3.5
 
+python.use_system_site_packages
+```````````````````````````````
+
+* Default: ``false``
+* Type: Boolean
+
+When true, it gives the virtual environment access to the global site-packages directory.
+Depending on the :ref:`yaml-config:build.image`,
+Read the Docs includes some libraries like scipy, numpy, etc.
```

Comment:
Same as #4419 

If we don't want / can't document exactly what are those libraries, I think we should at least link to the repository that contains the `Dockerfile` and tell this to the user. I mean, how they can do to know exactly what are the pre-installed libs of the docker image they are using.

Reply:
Maybe is worth having a `build-images` docs? (or creating docs in the docker images repo)

Reply:
I like the idea of having a page (inside the current docs) for the docker images that we use.

This page could explain what's the default image, explain or release process, what's installed on each of them, what's its base OS and how to check what specific version is installed on it.

Among other things 😁

Reply:
Well, I just discovered that we already have some docs about the packages installed, we should just improve/update them https://docs.readthedocs.io/en/latest/builds.html#packages-installed-in-the-build-environment

Reply:
Good! So, in that case, I'd say that we should link that documentation page from here. Seems to be the most accurate to do and easy to follow.

Reply:
We may need a different section for Python packages in those docs, though.

Reply:
Already did that haha, I thought you already saw the PR #4515